spec/features/admin_payroll_runs_spec.rb

```diff
@@ -91,6 +91,45 @@
     expect(page).to have_content payroll_run.claims.count
     expect(page).to have_content "Downloaded No"
     expect(page).to have_field("payroll_run_download_link", with: new_admin_payroll_run_download_url(payroll_run))
+
+    expect(page).to have_content payroll_run.payments[0].id
+    expect(page).to have_content payroll_run.payments[1].id
+
+    expect(page).to have_link(href: remove_admin_payroll_run_payment_path(id: payroll_run.payments[0].id, payroll_run_id: payroll_run.id))
+    expect(page).to have_link(href: remove_admin_payroll_run_payment_path(id: payroll_run.payments[1].id, payroll_run_id: payroll_run.id))
+  end
+
+  scenario "Service operator cannot see removal buttons when confirmation report has been uploaded" do
```

Comment:
I think this scenario is probably redundant because the ` "Service operator can remove a payment from a payroll run"` wouldn't work unless those remove links are on the page.

Reply:
This is testing the reverse now. That said, there are links, not buttons now, and the spec still passes, so maybe it's not worth having now 😆